projects/ngx-agora/src/lib/data/types/video-profile.type.ts

```diff
@@ -1,20 +1,30 @@
 /**
  * Video profiles to set an Agora.io Stream that will affect resolution, fps, and bitrate.
+ *
+ * @remark
+ * - Whether 1080 resolution or above can be supported depends on the device.
+ *   If the device cannot support 1080p, the actual frame rate is lower than the one listed in the table.
+ *   Agora optimizes the video on low-end devices.
+ * - With the update of web browsers, this table might not reflect all the supported profiles for each browser.
+ *   The actual support is subject to the device and web browser version.
+ * - Some versions of some web browsers might not support all the video profiles listed in the table.
+ *   In this case, we recommend you use the mainstream video profiles (the ones with the _1 suffix in the above table).
+ * - The Safari browser does not support modifying the video frame rate (30 fps by default).
+ *   If you set a frame rate other than 30 fps on Safari, the browser may change or reject your setting.
+ * - Due to limitations of some devices and browsers, the resolution you set may fail to take effect and get adjusted by the browser.
+ *   In this case, billings are calculated based on the actual resolution.
+ *
  * @see [Table](https://docs.agora.io/en/Video/API%20Reference/web/interfaces/agorartc.stream.html#setvideoprofile) for more details.
  */
 export type VideoProfile =
-  | '120p'
```

Comment:
@digitallysavvy why were these resolution settings removed from [the docs](https://docs.agora.io/en/Video/API%20Reference/web/interfaces/agorartc.stream.html#setvideoprofile)?

Reply:
I'm not sure why they were removed, these profiles may no longer exist in v3.0 of The Agora SDK. given 3.0 was a major version its very possible this was a breaking change that was made during the update.

Reply:
I'll go by what the docs say, but that definitely should be documented somewhere clear 😆